templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -47,28 +49,38 @@ contents:
           set -euxo pipefail
 
           {{if .Images.clusterEtcdOperatorImageKey}}
-            COMMAND="cluster-etcd-operator mount"
+            SERVER_COMMAND="cluster-etcd-operator mount"
+            PEER_COMMAND="cluster-etcd-operator mount"
+            METRIC_COMMAND="cluster-etcd-operator mount"
           {{else}}
-            COMMAND="kube-client-agent request"
+            COMMAND="kube-client-agent request \
+                --dnsnames={{etcdServerCertDNSNames .}} \
+                --commonname=system:etcd-server:${ETCD_DNS_NAME} \"
+            SERVER_COMMAND="$COMMAND
+                --ipaddrs=${ETCD_IPV4_ADDRESS},127.0.0.1 \"
```

Comment:
these need to add --dnsnames flags

Reply:
basically, if we are not using CEO rendered spec should look the same as it does today.